Installation error with healpix installed

I'm unable to build healpy on my OSX 10.10 laptop, with healpix and cfitsio installed via homebrew.

Error is below:

```
   building 'healpy._healpy_sph_transform_lib' extension
    clang -fno-strict-aliasing -fno-common -dynamic -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/usr/local/lib/python2.7/site-packages/numpy/core/include -I/usr/local/Cellar/cfitsio/3.370/include -I/usr/local/Cellar/healpix/3.11/include/healpix_cxx -I/usr/local/Cellar/cfitsio/3.370/include -I/usr/local/include -I/usr/local/opt/openssl/include -I/usr/local/opt/sqlite/include -I/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c healpy/src/_healpy_sph_transform_lib.cc -o build/temp.macosx-10.10-x86_64-2.7/healpy/src/_healpy_sph_transform_lib.o
    In file included from healpy/src/_healpy_sph_transform_lib.cc:29:
    In file included from /usr/local/lib/python2.7/site-packages/numpy/core/include/numpy/arrayobject.h:4:
    In file included from /usr/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:17:
    In file included from /usr/local/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1804:
    /usr/local/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: "Using deprecated NumPy API, disable it by "          "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
    #warning "Using deprecated NumPy API, disable it by " \
     ^
    healpy/src/_healpy_sph_transform_lib.cc:923:27: error: no matching member function for call to 'imag'
              almalms[i](l,0).imag(0.0);
              ~~~~~~~~~~~~~~~~^~~~
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:86:8: note: candidate function not viable: requires 0 arguments, but 1 was provided
        T &imag() { return im; }
           ^
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:88:14: note: candidate function not viable: requires 0 arguments, but 1 was provided
        const T &imag() const { return im; }
                 ^
    healpy/src/_healpy_sph_transform_lib.cc:926:27: error: no matching member function for call to 'real'
              almalms[i](l,0).real(x);
              ~~~~~~~~~~~~~~~~^~~~
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:82:8: note: candidate function not viable: requires 0 arguments, but 1 was provided
        T &real() { return re; }
           ^
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:84:14: note: candidate function not viable: requires 0 arguments, but 1 was provided
        const T &real() const { return re; }
                 ^
    healpy/src/_healpy_sph_transform_lib.cc:948:31: error: no matching member function for call to 'real'
                  almalms[i](l,m).real(xr/sqrt_two);
                  ~~~~~~~~~~~~~~~~^~~~
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:82:8: note: candidate function not viable: requires 0 arguments, but 1 was provided
        T &real() { return re; }
           ^
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:84:14: note: candidate function not viable: requires 0 arguments, but 1 was provided
        const T &real() const { return re; }
                 ^
    healpy/src/_healpy_sph_transform_lib.cc:949:31: error: no matching member function for call to 'imag'
                  almalms[i](l,m).imag(xi/sqrt_two);
                  ~~~~~~~~~~~~~~~~^~~~
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:86:8: note: candidate function not viable: requires 0 arguments, but 1 was provided
        T &imag() { return im; }
           ^
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/xcomplex.h:88:14: note: candidate function not viable: requires 0 arguments, but 1 was provided
        const T &imag() const { return im; }
                 ^
    In file included from healpy/src/_healpy_sph_transform_lib.cc:34:
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/arr.h:190:33: warning: field 'stm' is uninitialized when used here [-Wuninitialized]
        arrT(tsize sz) : arr_ref<T>(stm.alloc(sz),sz), own(true) {}
                                    ^
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/arr.h:307:21: note: in instantiation of member function 'arrT<xcomplex<double>, normalAlloc__<xcomplex<double> > >::arrT' requested here
        arr(tsize sz) : arrT<T,normalAlloc__<T> >(sz) {}
                        ^
    /usr/local/Cellar/healpix/3.11/include/healpix_cxx/alm.h:95:32: note: in instantiation of member function 'arr<xcomplex<double> >::arr' requested here
          : Alm_Base(lmax_,mmax_), alm (Num_Alms(lmax,mmax)) {}
                                   ^
    healpy/src/_healpy_sph_transform_lib.cc:282:27: note: in instantiation of member function 'Alm<xcomplex<double> >::Alm' requested here
      Alm< xcomplex<double> > almIalm;
                              ^
    2 warnings and 4 errors generated.
    error: command 'clang' failed with exit status 1

    ----------------------------------------
Command "/usr/local/opt/python/bin/python2.7 -c "import setuptools, tokenize;__file__='/private/var/folders/cl/fjcd0k5x6ync0fwnzvs80q_w0000gn/T/pip-build-OfgqJ7/healpy/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/cl/fjcd0k5x6ync0fwnzvs80q_w0000gn/T/pip-yFFRgH-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/var/folders/cl/fjcd0k5x6ync0fwnzvs80q_w0000gn/T/pip-build-OfgqJ7/healpy
```

--- a/setup.py
+++ b/setup.py
@@ -409,7 +409,7 @@ else:
         'local_source': 'cfitsio',
         'supports_non_srcdir_builds': False}),
         ('healpix_cxx', {
-        'pkg_config_name': 'healpix_cxx',
+        'pkg_config_name': 'healpix_cxx >= 3.30.0',
         'local_source': 'healpixsubmodule/src/cxx/autotools'})
     ]
     extension_list = [pixel_lib, spht_lib, hfits_lib,

